### ISSUE TITLE
VSCODE delete everything on the server!

### ISSUE THREAD

**Issue by rik72c** [microsoft/vscode#79734]
Issue Type: <b>Bug</b>

CRITICAL ISSUE, when I tried to drag a folder shotcut (mistakenly) from explorer/FTP to vscode screen, IT DELETED EVERYTHING FROM THE FTP AND NO WAY TO RECOVER IT.

VS Code version: Code 1.37.1 (f06011ac164ae4dc8e753a3fe7f9549844d15e35, 2019-08-15T16:17:55.855Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz (4 x 2904)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_deferred_display_list: disabled_off<br>skia_renderer: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.89GB (2.57GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (5)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-intelephense-client|bme|1.2.1
php-debug|fel|1.13.0
php-docblocker|nei|2.0.1
format-html-in-php|rif|1.5.3
livereload|zii|0.2.5

</details>

**Comment by bpasero** [comment]
@rik72c what are the steps exactly?

**Comment by rik72c** [comment]
![image](https://user-images.githubusercontent.com/7495294/63700019-26785200-c84c-11e9-90c7-d21c826c5e2e.png)

I'm afraid I can't redo what I did at the moment, because my client just uploaded everything back to the server. But here is a brief screenshot. I dragged **a shortcut** from explorer to VSCode and it started everything in that folder with a popup "deleting.." right away.

**Comment by rik72c** [comment]
[err_w_vs.zip](https://github.com/microsoft/vscode/files/3541737/err_w_vs.zip)

Apparently it happened again just now with normal file drag and drop. (Luckily i had them backed up this time.) Here is a video screen recording of how it happened.

At first I dragged a file onto the desktop and it worked normally. Then I dragged it into VSCode then the file is gone from the ftp, one by one.

Last time, as I reported, I dragged the whole folder then I could see the"deleting.." popup before I cancelled the process and found out many files has been deleted.
